go.mod

```diff
@@ -5,17 +5,18 @@ require (
 	github.com/btcsuite/btcd/btcutil v1.1.5
 	github.com/btcsuite/btcd/chaincfg/chainhash v1.1.0
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
-	github.com/grpc-ecosystem/grpc-gateway/v2 v2.5.0
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.22.0
 	github.com/jessevdk/go-flags v1.4.0
+	github.com/lightninglabs/faraday/frdrpc v0.0.0-00010101000000-000000000000
 	github.com/lightninglabs/lndclient v0.17.4-1
 	github.com/lightningnetwork/lnd v0.17.4-beta
 	github.com/lightningnetwork/lnd/cert v1.2.2
 	github.com/lightningnetwork/lnd/kvdb v1.4.4
 	github.com/shopspring/decimal v1.2.0
 	github.com/stretchr/testify v1.8.4
 	github.com/urfave/cli v1.22.9
-	google.golang.org/grpc v1.59.0
-	google.golang.org/protobuf v1.31.0
+	google.golang.org/grpc v1.65.0
```

Comment:
Hmm, I'm not sure if this update might bite us, since in lnd we have an older version. Or is the plan to update that as well?